### PR TITLE
Add Network Service dev doc

### PR DESCRIPTION
**Pull request by rezhil-vmw**

### CODE REVIEW

File: network-service/README.md [hunk 1]
```diff
@@ -57,11 +24,104 @@ The UI allows users to execute all necessary workflows:
...
+
+To view the resource YAML in the YAML Explorer panel at bottom.
+
+![NetworkService View_YAML](source/network-services-view-yaml.png "Network Services View YAML")
 
 ## List all VM Load Balancers
 
-### Actions
-### Details
+List of available Network Services. You can deploy and manage service in a self-service way using Kubernetes APIs.
```
Comment (kushagram-vmw) [inline network-service/README.md]: This should say "List of available Load Balancers. You can deploy and manage load balancers in a self-service way using Kubernetes APIs" correct?

File: network-service/README.md [hunk 2]
```diff
@@ -57,11 +24,104 @@ The UI allows users to execute all necessary workflows:
...
+Create VM Load Balancer workflow.
+
+On Create button click VM Load Balancer model open with pre-populated name. 
+We are able to edit the name, depend on the name 'Selector' value get populated.
+
+![Create_VM_Load_Balancer](source/vm-lb-create.png "Create VM Load Balancer")
+
+##### Create new port
+
+To add new port to port group. THe user will enter following required fields and click Add button.
```
Comment (kushagram-vmw) [inline network-service/README.md]: THe => The
Suggestion: "To add new port to _a_ port group, the user must enter the following fields and click Add button,"

File: network-service/README.md [hunk 3]
```diff
@@ -57,11 +24,104 @@ The UI allows users to execute all necessary workflows:
...
+
+![NetworkService View_YAML](source/network-services-view-yaml.png "Network Services View YAML")
 
 ## List all VM Load Balancers
 
-### Actions
-### Details
+List of available Network Services. You can deploy and manage service in a self-service way using Kubernetes APIs.
+
+Users can view all the Network Services created in the namespace. The user will see the following information in the grid's default view:
```
Comment (rrick-vmw) [inline network-service/README.md:67]: Same comment here : Network Services => Load Balancers

File: network-service/README.md [hunk 4]
```diff
@@ -57,11 +24,104 @@ The UI allows users to execute all necessary workflows:
...
+Click on the Service name to see all the details for the resource, including recent events.
+
+![VM_Load_Balancer Details](source/vm-lb-details.png "VM Load Balancer Details")
+
+### Action Menubar
+#### Create
+
+Create VM Load Balancer workflow.
+
+On Create button click VM Load Balancer model open with pre-populated name.
```
Comment (rrick-vmw) [inline network-service/README.md]: When you click the Create button, the VM Load balancer modal will open with a pre-populated name.

File: network-service/README.md [hunk 5]
```diff
@@ -57,11 +24,104 @@ The UI allows users to execute all necessary workflows:
...
+
+![VM_Load_Balancer Details](source/vm-lb-details.png "VM Load Balancer Details")
+
+### Action Menubar
+#### Create
+
+Create VM Load Balancer workflow.
+
+On Create button click VM Load Balancer model open with pre-populated name. 
+We are able to edit the name, depend on the name 'Selector' value get populated.
```
Comment (rrick-vmw) [inline network-service/README.md]: The name can be updated. When updating the name, the 'Selector' value will get updated accordingly.

File: network-service/README.md [hunk 6]
```diff
@@ -57,11 +24,104 @@ The UI allows users to execute all necessary workflows:
...
+
+![Create_VM_Load_Balancer_Valid](source/vm-lb-create-vaild.png "Create VM Load Balancer Valid")
+
+### Action Menubar
+
+![VM_Load_Balancers_Action_Menu](source/vm-lb-action-menu.png "VM Load Balancers Action Menu")
+
+#### View YAML
+
+To view the resource YAML in the YAML Explorer panel at bottom.
```
Comment (rrick-vmw) [inline network-service/README.md]: The panel can be at the right as well.

See the branch here for an example set of content for the grid action menu: https://github.com/vsphere-tmm/vsphere-with-tanzu-dev-center/blob/rrick/vmtkgupdates_content_images/vm-service/README.md

File: network-service/README.md (hunk 1 above)
Comment (rezhil-vmw) [reply network-service/README.md]: yes

File: network-service/README.md [hunk 7]
```diff
@@ -57,11 +24,115 @@ The UI allows users to execute all necessary workflows:
...
+![NetworkService Details](source/netwrok-service-details.png "Network Service Details")
+
+### Action Menubar
+#### View YAML
+
+To view the resource YAML in the YAML Explorer panel at bottom.
+
+![NetworkService View_YAML](source/network-services-view-yaml.png "Network Services View YAML")
 
 ## List all VM Load Balancers
```
Comment (fstrudel) [inline network-service/README.md]: I'm confused about this section. It seems about load balancers in general but title and screenshots are about *VM* load balancers only?

Comment (rezhil-vmw) [reply network-service/README.md]: In the network plugin, we list all the virtualmachineservices under services  and under load balncer tab, we list the vm Load balancers.

**Review by rrick-vmw — APPROVED** [review]

File: network-service/README.md (hunk 7 above)
Comment (rezhil-vmw) [reply network-service/README.md]: Updated the document title with generic LB. Will update the screen shot after making code change

**Pull request merged**